lib/applications/schema.ts

```diff
@@ -750,6 +752,8 @@ export default gql`
     mobilityAids: [MobilityAid!]
     otherPatientCondition: String
     temporaryPermitExpiry: Date
+    permitType: PermitType!
```

Comment:
I think `permitType` needs to be included to allow users to change the `permitType` using the `EditModal` but it seems to have been purposely omitted here (there's a comment on line 748 saying to omit it).

<img width="569" alt="image" src="https://user-images.githubusercontent.com/51224641/167758914-64057815-46f6-4557-985b-1971ff5f1e4c.png">

Reply:
Would you mind clarifying here? Is there anything I should look out for 👀

Reply:
@OustanDing bump^

Reply:
Yes, to clarify, your change here is correct, so no further action is needed in this file. What I meant by backwards compatibility is that we should ensure that any other API calls that depend on this input schema should still work.

Reply:
Sweet, tested the api the relevant api calls and everything looks good!